modules/default.sh

```diff
@@ -49,6 +48,7 @@ function default_network {
   rm -rf /etc/network/up-bridge.sh
   rm -rf /etc/network/eth0-shared.sh
   rm -rf /etc/network/mode
+  config delete WIFICOUNTRY
```

Comment:
If we are trying to set `WIFICOUNTRY` back to default, shouldn't we be setting it to `WIFICOUNTRY="US"` instead of deleting it?

Reply:
@dogi doesn't want it in the config file at all after running default network. The default values for variables are stored in the ./cli.sh file itself. Not in the config file. The config file basically stores any custom values set by users and overrides the defaults in ./cli.sh